.github/workflows/build.yml

```diff
@@ -153,15 +156,21 @@ jobs:
           ./tools/run_constructor.sh
 
       - name: Check installer signature (macOS)
-        if: ${{ runner.os == 'macOS' && github.event_name != 'pull_request' }}
+        if: ${{ runner.os == 'macOS' }}
         run: |
           # Installer package
           set -eo pipefail
-          pkgutil --check-signature ${MNE_INSTALLER_NAME}
+          if [[ "${{ github.event_name }}" != "pull_request" ]]; then
+            pkgutil --check-signature ${MNE_INSTALLER_NAME} || exit 1
```

Comment:
FYI @hoechenberger this is the only step that *fails* when in PR mode, so I removed the conditinal in the GH Actions step `if` above and put it here instead. The other adjustments below are just path failures from #195